llvm/lib/Target/RISCV/RISCVSubtarget.h

```diff
@@ -86,7 +86,6 @@ class RISCVSubtarget : public RISCVGenSubtargetInfo {
   RISCVInstrInfo InstrInfo;
   RISCVRegisterInfo RegInfo;
   RISCVTargetLowering TLInfo;
-  SelectionDAGTargetInfo TSInfo;
```

Comment:
This is a full object. It's not a pointer. It will be default constructed

Reply:
Yes you are right. My bad. Will close this issue. All tests passed without it though.